dspace/config/DryadJournalSubmission.testing.properties

```diff
@@ -120,6 +120,11 @@ journal.BJJ.parsingScheme = manuscriptCentral
 journal.BJJ.integrated=true
 journal.BJJ.allowReviewWorkflow=false
 journal.BJJ.publicationBlackout=true
+journal.bmjOpen.subscriptionPaid = true
```

Comment:
This line (and all others like it) needs to have the correct journal abbreviation -- in this case BJJ